File: src/main/java/com/lashstudio/entity/Cliente.java
package com.lashstudio.entity;

import java.util.UUID;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "Cliente")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Cliente {

    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    @Column(name = "ID_Cliente")
    private UUID id;

    @Column(name = "Nombre", nullable = false, length = 50)
    private String nombre;

    @Column(name = "Apellido_Paterno", length = 50)
    private String apellidoPaterno;

    @Column(name = "Apellido_Materno", length = 50)
    private String apellidoMaterno;

    @Column(name = "Telefono", length = 10)
    private String telefono;

    @Column(name = "Correo", unique = true, nullable = false, length = 50)
    private String correo;

    @Column(name = "Password", nullable = false, length = 100)
    private String password;
}

File: src/models/Cliente.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/database');

const Cliente = sequelize.define('Cliente', {
    id: {
        type: DataTypes.UUID,
        defaultValue: DataTypes.UUIDV4,
        primaryKey: true,
        field: 'ID_Cliente' // Coincide con @Column(name = "ID_Cliente")
    },
    nombre: {
        type: DataTypes.STRING(50),
        allowNull: false,
        field: 'Nombre'
    },
    apellidoPaterno: {
        type: DataTypes.STRING(50),
        field: 'Apellido_Paterno'
    },
    apellidoMaterno: {
        type: DataTypes.STRING(50),
        field: 'Apellido_Materno'
    },
    telefono: {
        type: DataTypes.STRING(10),
        field: 'Telefono'
    },
    correo: {
        type: DataTypes.STRING(50),
        allowNull: false,
        unique: true,
        field: 'Correo',
        validate: {
            isEmail: true
        }
    },
    password: {
        type: DataTypes.STRING(100),
        allowNull: false,
        field: 'Password'
    }
}, {
    tableName: 'cliente', // Coincide con @Table(name = "Cliente")
    timestamps: false    // En Java no tenías createdAt/updatedAt
});

module.exports = Cliente;